aries-site/src/pages/components/search.js

```diff
@@ -0,0 +1,193 @@
+          <Box width={{ max: 'large' }}>
+            <BulletedList
+              level={3}
+              items={[
+                <Text>
+                  Provide effective{' '}
+                  <Anchor href="#search-with-auto-suggestions">
+                    auto-suggestions
+                  </Anchor>{' '}
+                  whenever possible.
+                </Text>,
+                `Auto suggestions accelerate search and allow users to focus
+                on task completion.`,
+              ]}
+            />
+          </Box>
+        </Subsection>
+        <Subsection name="Returning search results" level={3} gap="small">
+          <Box width={{ max: 'large' }}>
+            <BulletedList
+              level={3}
+              items={[
+                `Use "live filtering." When results are visibile and quick to 
+                filter, results should be filtered as the user types.`,
+                `Sort results by highest relevance. Relevance can be 
+                determined by keyword matching, however, to really elevate user 
+                experience, consider implementing probability based 
+                approaches.`,
+                `Persist users' original query. This eliminates the need to 
+                re-type and assists query reformation if needed.`,
+                `Persist users' recently used search queries. Remembering the 
+                user's previous efforts respects their time and serves as 
+                "short cuts" for returning to previously visited content.`,
+              ]}
+            />
+          </Box>
+        </Subsection>
+        <Subsection name="Accessibility" level={3} gap="small">
+          <SubsectionText>
+            Search inputs should be accessible. Ensure users have the ability to
+            return results using their keyboard as well as clicking a button.
+          </SubsectionText>
+          <SubsectionText>
+            Inputs should always be accompanied by labels for accessibility. An
+            icon may serve as a label if: 1) the icon meaning is well
+            understood, and 2) has an 'aria-labelledby' attribute. For
+            additional detail, see{' '}
+            <Anchor
+              href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              ARIA form hints
+            </Anchor>{' '}
+            and{' '}
+            <Anchor
+              href="https://www.w3.org/WAI/tutorials/forms/labels/#using-aria-labelledby"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              'aria-labelledby'
+            </Anchor>
+            .
+          </SubsectionText>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Variants">
+          <SubsectionText>
+            Visual and functional extensions for Search.
```

Comment:
For variants maybe including that search can be used as an overall site level in the header or can be used in smaller spaces within a page